index.ts

```diff
@@ -300,19 +300,17 @@ commander
             q: 'mimeType="application/vnd.google-apps.script"',
           });
           const files = data.files;
-          const fileIds:string[] = [];
+          const fileIds: Array<{ name: string; value: string; }> = [];
           if (files.length) {
             files.map((file: any) => {
-              fileIds.push(`${file.name}`.padEnd(20) + ` - (${file.id})`);
+              fileIds.push({name: `${file.name}`.padEnd(20) + ` - (${file.id})`, value: `${file.id}`});
```

Comment:
I think this should be formatted on multiple lines.

Reply:
```ts
fileIds.push({
  name: `${file.name.padEnd(20)} - (${file.id})`,
  value: file.id,
});
```

Reply:
why is this not just
```ts
            const fileIds = files.map((file) => {
              return {
                name: `${file.name.padEnd(20)} - (${file.id})`,
                value: file.id
              };
            });
``` 
?
Typescript can figure out the type here, no need to explicitly state it.

Reply:
👍 This would be an even better refactor.

Reply:
Perfect, this has now been fixed.